connector-packager/connector_packager/xsd_validator.py

```diff
@@ -176,3 +178,13 @@ def validate_file_specific_rules(file_to_test: ConnectorFile, path_to_file: Path
                     return False
 
     return True
+
+# Check if connector file content contains warnings needs to notify connector developer
+def check_file_content(file_to_test: ConnectorFile, path_to_file: Path):
+
+    if file_to_test.file_type == 'dialect':
+        xml_tree = parse(str(path_to_file))
+        root = xml_tree.getroot()
+        if 'base' in root.attrib and root.attrib['base'] == 'DefaultSQLDialect':
+            logger.warning("Warning: DefaultSQLDialect is not a recommended base to inherit from, please change your base dialect to a better one, "
+                           "you can find the recommended base dialect list on our SDK documentation page.")
```

Comment:
Point to the specific page: https://tableau.github.io/connector-plugin-sdk/docs/design#choose-a-dialect
```suggestion
                           "you can find the recommended base dialect list here: https://tableau.github.io/connector-plugin-sdk/docs/design#choose-a-dialect.")
```

Reply:
Done, thanks!